src/transpiling/jsx_precompile.rs

```diff
@@ -230,6 +230,29 @@ fn get_attr_name(jsx_attr: &JSXAttr, is_component: bool) -> String {
   }
 }
 
+fn normalize_lit_str(lit: &Lit) -> Lit {
+  match lit {
+    Lit::Str(lit_str) => {
+      let value = lit_str.value.to_string();
```

Comment:
I think this will work to avoid a clone?

```suggestion
      let value: &str = &lit_str.value;
```